### DISCUSSION TITLE
[Feature Request]: Invite only

### DISCUSSION THREAD

**Discussion by nabeelio** [tryzealot/zealot#1474] [Q&A] [answered]
### 描述 | Description

I can't tell in the docs, is there a way to have users sign up through an invite link? Or is it 3rd party auth only and anyone can sign up?

**Comment by icyleaf** [comment, marked as answer]
By default, administrator could invite users in admin panel by create new user. sent e-mail via enabled STMP or you aslo review the invite link in created user detail page. (Missing the docs) And disable register mode https://zealot.ews.im/docs/self-hosted/configuration/environment-variables#enable-new-user-registration

and zealot support 3-party auth login https://zealot.ews.im/docs/self-hosted/configuration/third-party-authentication
